docker-compose.yml

```diff
@@ -3,8 +3,6 @@ sequelize:
   links:
    - mysql
    - postgres
-  volumes:
```

Comment:
I think volumes is used here to mount the local files so you don't have to rebuild the container each time a file changes - But @mickhansen knows more about that